File: nodeExpress/src/models/userModel.ts
import { Schema, model } from 'mongoose';

const userSchema = new Schema(
  {
    user_first_name: {
      required: true,
      type: String,
    },
    user_last_name: {
      required: true,
      type: String,
    },
    user_birth: {
      required: true,
      type: Date,
    },
    user_country: {
      required: true,
      type: String,
    },
  },
  {
    timestamps: true,
  }
);

export default model('User', userSchema);

File: JavaSpringBoot/src/main/java/com/rtxct/simple/schema/model/UserModel.java
package com.rtxct.simple.schema.model;

import java.util.Calendar;
import java.util.Date;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "tbl_user")
public class UserModel {

	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	@Column(name = "user_id")
	private Long userId;
	
	@Column(name = "user_first_name")
	private String userFirstName;
	
	@Column(name = "user_last_name")
	private String userLastName;
	
	@Column(name = "user_birth")
	private Date userBirth;
	
	@Column(name = "user_country")
	private String userCountry;
	
}
